inductiva/molecules/protein_solvation/README.md

```diff
@@ -39,7 +39,7 @@ To test this scenario we have available one example of a protein PDB file - [dow
 inductiva.api_key = "YOUR_API_KEY"
 
  # Initialize the scenario
- scenario = inductiva.molecules.scenarios.ProteinSolvation(
+ scenario = inductiva.molecules.ProteinSolvation(
```

Comment:
Is this how the code would look like with that previous import?

Reply:
Yes! It fits the coastal area scenarios that were moved by Fábio recently.

Reply:
ok!